imports/startup/both/collection-patches.js

```diff
@@ -75,7 +75,7 @@ Mongo.Collection.prototype._applyBehaviour = function _applyBehaviour(behaviour,
   const collection = this;
   collection.attachSchema(behaviour.schema, options);
   // TODO: Only 0 values supported in public fields
-  if (behaviour.publicFields) {
+  if (behaviour.publicFields && collection.publicFields) {
```

Comment:
Ha a collection-re nem raktak publicFields-et, de a behaviour-re igen, akkor eddig ilyenkor letrehozta a collection-on a publicFields-et, a behaviour-nek a publicFields-jebol. De mostantol ilyenkor nem fogja figyelembe venni a behaviour publicFields-jet sem. Biztos hogy ezt akartad?

Reply:
Ha nincs a kollekción publicFields, akkor az összes mező automatikusan publikus. Ha a behaviour ráteszi a sajátját, akkor csak az lesz a publikus, az összes többit buktuk.

Reply:
Az lenne a tuti, hogy ha eddig nem volt a kollekción publicFields és a behaviourön meg van, akkor tegye bele a kollekció összes mezőjét is. Legyen így?

Reply:
Az megoldas lehetne. De a kollekciotol hogy lehet elkerni az osszes mezot?

Reply:
Masik zavar az eroben, a jelentesekkel kapcsolatban, hogy akkor most ha pl egy behaviour nem ad meg publicFields-t, az azt jelenti, hogy minden field-je publikus? Mert ezt nem igy kezeljuk. Vagy teljesen maskepp ertelmezzuk ott?

Reply:
Úgy tűnik ez nem lett összeegyeztetve, nincs egységes rendszer. Most úgy van, hogy ha a behaviour nem ad meg publicFields-et, akkor ha olyan kollekcióra kerül, ahol szintén nincs megadva, akkor minden fieldje publikus. Ha meg olyan kollekcióra, ahol van megadva, akkor elvileg egyik fieldje sem lesz publikus, de ügyesen megkerüli ezt pl. a topic-timestamp, ahol a topic publicFieldsei közé van beírva a createdAt és társai. Ha meg van megadva behaviournél publicFields, eddig mindig olyan kollekcióra került, aminek szintén volt, így nem okozott eddig gondot.

Reply:
Hat igen. Most hogy igy mondod Totyanak ki volt adva hf-be hogy probaljon erre kitalalni egy jo megoldast. De aztan mas teendok mellett nem jutott ra ideje.